onyx/onyx_sram_subsystem/mock_mem.py

```diff
@@ -0,0 +1,328 @@
+import magma as m
+
+# I am not making the read latency a generator param
+# as it is not parameter of Chisel provided by Gedeon
+READ_LATENCY: int = 1
+
+class SRAMBase(m.Generator2):
+    # *args, **kwargs for inheritance reasons
+    def __init__(self,
+            addr_width: int,
+            data_width: int,
+            has_byte_enable: bool,
```

Comment:
```suggestion
            has_byte_enable = False: bool,
```

Since we aren't supporting True yet